flashinfer/gemm/gemm_mm_mxfp8_cute_dsl.py

```
@@ -0,0 +1,291 @@
+            expected_m=m,
+        )
+
+    class B12xMxfp8GemmRunner(TunableRunner):
+        """TunableRunner for b12x block-scaled MXFP8 dense GEMM on SM12x."""
+
+        def get_valid_tactics(
+            self,
+            inputs: List[torch.Tensor],
+            profile: OptimizationProfile,
+        ) -> list:
+            (a, b, a_descale, b_descale, _, out, _) = inputs
```

Ruff (0.16.1)

[warning] Unpacked variable `a_descale` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)